dbt_copilot_helper/utils/files.py

```diff
@@ -1,28 +1,133 @@
 from pathlib import Path
 
 import click
+import yaml
+from schema import Optional
+from schema import Or
+from schema import Schema
 
+from dbt_copilot_helper.utils.validation import validate_string
 
-def mkdir(base, path):
-    if (base / path).exists():
-        return f"Directory {path} exists; doing nothing"
+range_validator = validate_string(r"^\d+-\d+$")
+seconds_validator = validate_string(r"^\d+s$")
 
-    (base / path).mkdir(parents=True)
-    return f"Directory {path} created"
+BOOTSTRAP_SCHEMA = Schema(
```

Comment:
As the schemas are related to validation rather than directly with files, might be better in the `dbt_copilot_helper.utils.validation` module?